estuary.log

```diff
@@ -0,0 +1,48 @@
+2022/04/14 16:10:08 failed to sufficiently increase receive buffer size (was: 208 kiB, wanted: 2048 kiB, got: 416 kiB). See https://github.com/lucas-clemente/quic-go/wiki/UDP-Receive-Buffer-Size for details.
```

Comment:
@gmelodie this shouldn't be committed

Reply:
this file is still here, need to delete it and rebase it out of the history so we dont end up with it in the repo